src/main/java/com/mitchej123/hodgepodge/mixins/early/minecraft/MixinInventoryCrafting.java

```diff
@@ -0,0 +1,32 @@
+package com.mitchej123.hodgepodge.mixins.early.minecraft;
+
+import net.minecraft.inventory.Container;
+import net.minecraft.inventory.IInventory;
+import net.minecraft.inventory.InventoryCrafting;
+import net.minecraft.item.ItemStack;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Shadow;
+
+@Mixin(InventoryCrafting.class)
+public class MixinInventoryCrafting {
+
+    @Shadow
+    private ItemStack[] stackList;
+
+    @Shadow
+    private Container eventHandler;
+
+    /**
+     * @author kuba6000
+     * @reason Optimize InventoryCrafting
+     */
+    @Overwrite
```

Comment:
I think this would be better as a HEAD inject and cancel rather than an overwrite